### PR TITLE
CE-3295: Display upvotes count on RWA

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
## Links

* https://wikia-inc.atlassian.net/browse/CE-3295

## Description

We added possibility to upvote user edit/change. This code displays number of upvotes for each revision on Recent Wiki Activity page.

## Reviewers

@Wikia/community-engineering @Wikia/x-wing

### CODE REVIEW

File: front/main/app/templates/components/recent-change.hbs [hunk 1]
```diff
@@ -39,6 +39,9 @@
 		<strong class="recent-change__summary-label">{{i18n 'main.summary' ns='recent-wiki-activity'}}</strong> {{{model.parsedcomment}}}
 	</small>
 {{/if}}
+<div class="recent-change__upvotes{{if userUpvoted ' user-upvoted'}}">
```
Comment (bognix) [inline front/main/app/templates/components/recent-change.hbs]: minor: consider adding space before `if` not inside class name string in `if` helper

File: front/main/app/components/recent-change.js [hunk 2]
```diff
@@ -3,9 +3,18 @@ import Ember from 'ember';
...
 	active: Ember.computed('id', 'rc', function () {
 		return this.get('id') === this.get('rc');
 	}),
+	userUpvoted: Ember.computed('model.upvotes', 'currentUser.userId', function () {
+		const upvotes = this.get('model.upvotes');
+
+		if (upvotes) {
+			return upvotes.isAny('from_user', this.get('currentUser.userId').toString());
+		}
+		return false;
```
Comment (bognix) [inline front/main/app/components/recent-change.js]: minor:
`return upvotes && upvotes.isAny(...)`

**Comment by bognix** [comment]
Two minor issues that might be considered
LGTM! :+1:

**Comment by vforge** [comment]
LGTM, just merge `dev` to fix the tests.

**Comment by Warkot** [comment]
LGTM

**Comment by Warkot** [comment]
Hey, this PR is over three weeks old. Therefore, something should be done with this slacker—probably merged or closed and, after one of the aforementioned, the branch deleted to keep the repo clean.

**Comment by lukaszkonieczny** [comment]
Thanks Warkot! This PR is blocked by other work, which was postponed because of ... other work :) But we will merge today really soon.

**Comment by Warkot** [comment]
Awesome! :smile_cat: :+1:

**Comment by kamilkoterba** [comment]
Moved here #2204

**Pull request closed without merging**